services/movie_session.py

```diff
@@ -21,14 +25,14 @@
 
 
 def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+    return get_object_or_404(MovieSession, id=movie_session_id)
 
 
 def update_movie_session(
-    session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+        session_id: int,
+        show_time: str = None,
```

Comment:
Consider using `Optional[datetime]` for `show_time` to explicitly indicate that this parameter can be `None` and to ensure proper handling of date and time formats.

Reply:
This has already been written by the mentor